KubernetesExternalClients/java/src/main/java/KubernetesExternalTest.java

```diff
@@ -0,0 +1,59 @@
+import com.hazelcast.client.HazelcastClient;
+import com.hazelcast.client.config.ClientConfig;
+import com.hazelcast.client.properties.ClientProperty;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.map.IMap;
+import com.hazelcast.spi.properties.HazelcastProperty;
+
+import java.util.Random;
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.logging.Logger;
+
+/**
+ * Kubernetes external test for thin java client
+ */
+public class KubernetesExternalTest {
+
+    private static final Logger LOGGER;
+    static {
+        // The following makes jdk logging appear in single lines instead of 2 lines:
+        // https://stackoverflow.com/a/10706033
+        System.setProperty("java.util.logging.SimpleFormatter.format",
+                "%1$tY-%1$tm-%1$td %1$tH:%1$tM:%1$tS %4$s %2$s %5$s%6$s%n");
+        LOGGER = Logger.getLogger(KubernetesExternalTest.class.getName());
+    }
+
+    public static void main( String[] args ) {
+        String externalIp = "<EXTERNAL-IP>";
+
+        ClientConfig clientConfig = new ClientConfig();
+        clientConfig.getNetworkConfig().addAddress(externalIp);
+        // disable client logging
+        clientConfig.setProperty( "hazelcast.logging.type", "none" );
```

Comment:
why?

Reply:
https://github.com/hazelcast/client-compatibility-suites/pull/128/commits/50929269b589761da39d140701537d13a2341cea

Reply:
Instead of doing this, we could have written a more test specific log like `KubernetesExternalTest is successfull since the client is connected successfully!` . Then we grep this line in the verification step:
          
```
kubectl logs pod/hazelcast-${{ matrix.client-language }}-client | grep 'KubernetesExternalTest is successfull since the client is connected successfully!' -q
```

And we do not need to disable the logs, right?

Reply:
Yeah makes sense, we should make sure no error is printed. Let me refactor the test so that they end after 20 operations and print some messages like that. And if an error happens, they should not print it, and print the error instead.

Reply:
I've applied it https://github.com/hazelcast/client-compatibility-suites/pull/128/commits/5bbcf39ff3c476f6585a3b8e1fc7a3aa60bd3050

For internal test, I don't want to do it because 

1. I need to change all client tests because github actions workflow uses single assertion logic for all clients. (I can create another verify for 'java' and make it different from other clients, however it's not elegant)
2. Internal client tests don't end and run indefinitely. Checking the last 20 sentence makes more sense for internal client tests.

Reply:
https://github.com/hazelcast/client-compatibility-suites/actions/runs/10717704093/job/29718159903

Reply:
applied https://github.com/hazelcast/client-compatibility-suites/pull/128/commits/267a1fe43e0c9331b3508424f843b0aeec2c1522 and rerun:  https://github.com/hazelcast/client-compatibility-suites/actions/workflows/KubernetesInternalClients.yaml